### PR TITLE
Add getConfigFileContent protocol for MC [MC-2995]

### PR DESCRIPTION
**Pull request by emrberk**
Monorepo PR: https://github.com/hazelcast/hazelcast-mono/pull/3241

### CODE REVIEW

**Review by srknzl — COMMENTED** [review]
I see that 5.5 used 2.8, is this change for 5.5? Or 6.0? 

see https://github.com/hazelcast/hazelcast-mono/pull/2307 and https://github.com/hazelcast/hazelcast-client-protocol/pull/527/files#diff-bb149721c03e94160908a744c251592019df822dddde9495c7c1532be39d55de

6.0 should be 2.9. Please verify

**Comment by srknzl** [comment]
Can u link the main PR to this?

**Comment by emrberk** [comment]
> I see that 5.5 used 2.8, is this change for 5.5? Or 6.0?
> 
> see [hazelcast/hazelcast-mono#2307](https://github.com/hazelcast/hazelcast-mono/pull/2307) and https://github.com/hazelcast/hazelcast-client-protocol/pull/527/files#diff-bb149721c03e94160908a744c251592019df822dddde9495c7c1532be39d55de
> 
> 6.0 should be 2.9. Please verify

MC 5.6 will use Platform 5.5.2, so this change will be for 5.5.2 and onwards. I assume having 2.8 is correct here? 🤔

**Comment by emrberk** [comment]
> Can u link the main PR to this?

Linked, thanks.

**Review by srknzl — APPROVED** [review]

File: protocol-definitions/MC.yaml [hunk 1]
```diff
@@ -965,3 +965,19 @@ methods:
...
+      retryable: true
+      partitionIdentifier: -1
+    response:
+      params:
+        - name: configFileContent
+          type: String
+          nullable: true
+          since: 2.8
+          doc: |
+            Member configuration file content (XML or YAML string) of the member.
```
Comment (ihsandemir) [inline protocol-definitions/MC.yaml:983]:
```suggestion
            Member configuration file content (XML or YAML string) of the member or null if the server was not started using a configuration file.
```

**Review by ihsandemir — APPROVED** [review]

**Pull request merged**